crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -306,7 +305,7 @@ impl Schedule {
     /// #
     /// let stage = schedule.get_stage::<SystemStage>(&"my_stage").unwrap();
     /// ```
-    pub fn get_stage<T: Stage>(&self, label: &dyn StageLabel) -> Option<&T> {
+    pub fn get_stage<T: Stage>(&self, label: StageLabelId) -> Option<&T> {
```

Comment:
The only reason these fns originally used dynamic dispatch is likely due to [this issue](https://github.com/rust-lang/rust/issues/83701). For now we can just accept `StageLabelId`, then switch it to `impl StageLabel` once rust 1.63 comes out.